build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,171 @@
+$ErrorActionPreference = "Stop"
+$OVSVersion = "3.0.5-antrea.0"
+$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$OVSVersion-win64.zip"
+$OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+$WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+$OVSDownloadDir = $WorkDir
+$OVSInstallDir = "C:\openvswitch"
+$OVSZip = "$OVSDownloadDir\ovs-win64.zip"
+$ImportCertificate = $true
+$CheckFileHash = $true
+$PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+function WaitExpandFiles($Src, $Dest) {
+    Write-Host "Extract $Src to $Dest"
+    Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+}
+
+function CheckIfOVSInstalled() {
```

Comment:
If the purpose is to check if OVS driver is installed or not, you can use this powershell command "netcfg.exe -q ovsext". I didn't get your point about checking the existence of $OVSInstallDir inside container. The required OVS userspace files are existing inside the container. And did you try if it works the driver files only existing inside the host process container?

Reply:
This function checks if OVS directory is present on host or not , OVSInstallDir is C:/openvswitch

Reply:
In pristine host case, openvswitch directory is not required to be present initially, you can get the openvswitch driver files and other required file from image and install driver and next time check only for driver presence, for all required ovs files, only one source, image should be used.

Reply:
I have removed the use of openvswitch directory on host